src/Lib/SmtLang/Smt.hs

```diff
@@ -0,0 +1,130 @@
+module Lib.SmtLang.Smt
+    (
+    ) where
```

Comment:
Basically it's not finished :p

On Thu, 1 Sep 2016, 05:16 Michael Gale, <notifications@github.com> wrote:

> In src/Lib/SmtLang/Smt.hs
> <https://github.com/zetty-z3/zetty/pull/4#discussion_r77111802>:
>
> > @@ -0,0 +1,130 @@
> > +module Lib.SmtLang.Smt
> > +    (
> > +    ) where
>
> Is it intentional that this module does not export anything?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/zetty-z3/zetty/pull/4/files/1788e74f23a6b2fa5f8721c368844e1acc06bf99#r77111802>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ACCkUNUWmn-bqPhdRr0WR4fI3OgYHtzjks5qllGggaJpZM4Jw3qM>
> .
>